src/config/config.go

```diff
@@ -15,6 +15,7 @@ import (
 type Config struct {
 	Role                 string
 	ClusterID            string
```

Comment:
do we still need the 'ClusterID' ?

Reply:
yes, it is used for access the cluster and getting its hosts by assisted-controller